### DISCUSSION TITLE
In the ClothDemo,when use XPBD, distanceStiffness = 100000,Why?

### DISCUSSION THREAD

**Discussion by ColdNight115** [InteractiveComputerGraphics/PositionBasedDynamics#130] [Q&A] [answered]
the value range of stiffness is [0,1] with other method.
is this because of units of length?

**Comment by janbender** [comment, marked as answer]
This is because PBD does not use physically meaningful values and in theory simulates an infinitely stiff material (if you iterate until convergence). XPBD solves this problem and uses physically meaningful values.

**Reply by ColdNight115** [reply]
I found the source：
http://blog.mmacklin.com/2016/10/12/xpbd-slides-and-stiffness/
